PR-Trigger/index.js

```diff
@@ -168,16 +174,6 @@ const httpTrigger = async function (context, _req) {
         return;
     }
 
-    // Allow the state to declare that nothing should happen
-    if (state.type === "noop") {

```

Comment:
'"error" | "remove" | "info"' and '"noop"' have no overlap since https://github.com/DefinitelyTyped/dt-mergebot/pull/36/files#diff-4dfd8d72bb4721e407b799cd8841c7d018fa17f6b533456fd1a5b68e8ffc4587L44

Reply:
I ran into this in #317, so I removed it there.